src/css/organisms/filter-form.scss

```diff
@@ -56,6 +56,28 @@
       padding-bottom: 0px;
     }
   }
+
+  /* - Internet Explorer 10–11
+   - Internet Explorer Mobile 10-11 */
+  :-ms-input-placeholder {
+    font-size: 0.8rem;
+    padding: 4px;
+  }
+
+  .filter-form-search {
+    padding: $spacing-xs;
+
+    @media #{$mq-md} {
+      width: 100%;
+    }
+    width: 97%;
+    @supports (-webkit-touch-callout: none) {
```

Comment:
This was my way to target iOS Safari uniquely. What do you think?

Reply:
CSS feature detection FTW 😸